pkg/config/config.go

```diff
@@ -129,7 +129,21 @@ func (bp *Blueprint) Module(id ModuleID) (*Module, error) {
 		return nil
 	})
 	if mod == nil {
-		return nil, fmt.Errorf("%s: %s", errorMessages["invalidMod"], id)
+		err := InvalidModuleError{id}
+		clMod := ""
+		minDist := -1.0
+		bp.WalkModules(func(m *Module) error {
+			dist := float64(levenshtein.Distance(string(m.ID), string(id), nil)) / float64(len(string(m.ID)))
+			if minDist == -1.0 || dist < minDist {
+				minDist = dist
+				clMod = string(m.ID)
+			}
+			return nil
+		})
+		if clMod != "" && minDist < 0.6 {
```

Comment:
Let's not use ratio, but absolute distance (say 3).
E.g. if `m.ID` the ration will be always `>= 1.0`, unless it's a match.

Reply:
Done.Done.